redshells/app/__init__.py

```diff
@@ -0,0 +1 @@
+from redshells.contrib.app.word_item_similarity.calculate_similarity_with_matrix_factorization import CalculateSimilarityWithMatrixFactorization
```

Comment:
@hirosassa this seems a little bit tricky import style

1. I think it's better to import with full `import redshells.contrib.app.word_item_similarity.calculate_similarity_with_matrix_factorization`
path rather than `import redshells.app`

2. Another option is putting import statement inside of `SpecialClass.__init__` as pandas do https://github.com/pandas-dev/pandas/blob/059c8bac51e47d6eaaa3e36d6a293a22312925e6/pandas/io/excel/_xlrd.py#L24

Reply:
Thanks! fixed in c985cdf

Reply:
@Hi-king ping

Reply:
@hirosassa [ASK]
Why do we need aliasing here?
I think CalculateSimilarityWithMatrixFactorization can be put only in contrib like other models (e.g. FactorizationMachineGraph).


(Are there any special reason we can't write `redshells.contrib.app.word_item_similarity.calculate_similarity_with_matrix_factorization` in applications? )

Reply:
@Hi-king I found this is my mistake. This does not exist in original version (current `master` ).
I removed in 723a7f3

Reply:
@hirosassa 👍